recipes/eccodes/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
@kmuehlbauer I am not a Windows expert so let's leave it alone and open an issue once the `feedstock` is created. I usually ask @gillins for help :stuck_out_tongue_winking_eye: